tests/publiccloud/aistack_basic.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+# Summary: Basic publiccloud AIstack test
+#     create AWS sle-micro-6-0 vm
+#     Install RKE2 on the instance
+#     Check the system version is the expected one
+# Maintainer: Yogalakshmi Arunachalam yarunachalam@suse.com
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use File::Basename;
+
+sub run {
+    #Create aws instance
+    select_serial_terminal();
+    my $provider = $args->{my_provider};
+    my $instance = $provider->create_instance();
+    registercloudguest($instance) if is_byos();
+    register_addons_in_pc($instance);
+
+    #Update and install dependencies
+    $instance->ssh_assert_script_run( 'sudo zypper ref; sudo zypper -n up',
+        timeout => 1000 );
+    record_info('Refresh and update');
+    sleep 90;    # wait for a bit for zypper to be available
+
+    #Install Docker (required for RKE2)
+
+    $instance->ssh_assert_script_run( 'sudo zypper install -y curl',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo zypper install -y docker',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo systemctl start docker',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo systemctl enable docker',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo systemctl status docker',
+        timeout => 100 );
+    record_info('Installed Docker');
+
+    #Install RKE2
+    $instance->ssh_assert_script_run( 'curl -sfL https://get.rke2.io | sh',
+        timeout => 1000 );
+    $instance->ssh_assert_script_run( 'sudo systemctl enable rke2-server',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo systemctl start rke2-server',
+        timeout => 100 );
+    $instance->ssh_assert_script_run( 'sudo systemctl status rke2-server',
+        timeout => 100 );
+    record_info('Installed RKE2');
+
+    #Install kubectl
+    $instance->ssh_assert_script_run(
+        'curl -LO "https://dl.k8s.io/release/v1.26.0/bin/linux/amd64/kubectl',
```

Comment:
Should the version be hard-coded this way?

Reply:
This is draft for testing. Planning to do as a parameter.

Reply:
done